### PR TITLE
Update User Profile Responsive styles

### PR DESCRIPTION
**Pull request by jm90m**
* For medium/small screen sizes: keep username on separate lines, and centered.
* Trello: https://trello.com/c/PWPN1ZVS/255-responsive-user-hero-center-avatar-and-bring-name-in-another-line
* 
screen shot 2017-10-15 at 2 56 24 pm
responsive-user-page

### CODE REVIEW

File: src/styles/custom.less [hunk 1]
```diff
@@ -47,3 +47,7 @@
 @layout-trigger-height       : 48px;
 @layout-zero-trigger-width   : 36px;
 @layout-zero-trigger-height  : 42px;
+
+// Screen Sizes
+@medium-screen: 990px;
```
Comment (bonustrack) [inline src/styles/custom.less]: We already have existing value for screensize see here: https://github.com/busyorg/busy/blob/702bc7cb698fb8f200a768024cfbc0285258ad13/src/styles/modules/variables.less#L7-L9
I think we the responsive layout should be triggered when size is lower than `768px`. Similar than right sidebar or search in topnav.

**Comment by Sekhmet** [comment]
I feel like it would be more logical if `@handle` and user rank was rendered directly below username and above buttons. I'm afraid that it might require some refactoring of UserProfile to make this change though.
What do you think?

**Comment by jm90m** [comment]
@bonustrack should we keep username and user rank on same line?
@Sekhmet yeah I agree, should I refactor to move @handle and user rank above the action buttons?

**Comment by Sekhmet** [comment]
I think that it should look like this on mobile:
- Avatar
- Username
- Handle
- Rank
- Buttons

On desktop it should still look like it looks now.

**Comment by bonustrack** [comment]
I agree too, maybe also these 2 could be on same line:
image
What you think?

**Comment by Sekhmet** [comment]
Yeah, I think it would make sense to put those buttons on the same line.

**Comment by jm90m** [comment]
I like that order @Sekhmet 

@bonustrack so those are 2 action items that do 2 different things, I think its good to keep them in separate lines, since they do different actions. One takes you to a different page while another brings a menu up

**Comment by bonustrack** [comment]
@jm90m the button will mostly be "Follow" when not seeing your own profile and the menu [...] appear in down on small screen. IMO it look more sexy if the hero frame has not too much height.

**Comment by jm90m** [comment]
@bonustrack ahh I see makes sense then to keep them on same line

**Comment by bonustrack** [comment]
Menu [...] could also appear in top right like this:
image

File: src/styles/custom.less (hunk 1 above)
Comment (jm90m) [reply src/styles/custom.less]: we can't use these properties, these are set to be `min-width:` rather than `max-width`

**Comment by jm90m** [comment]
Just updated to move the menu to top right and move buttons down when using small screen
Also updated order based on what @Sekhmet said.
screen shot 2017-10-15 at 7 15 59 pm

**Comment by bonustrack** [comment]
- Red and blue line should be same size. Now border in top and in down is not same.
image
- Also there is too much space between name and @handle (see green line)
- I still think we need the @handle and Grade in same line

**Comment by jm90m** [comment]
Updated styling to be more aligned @bonustrack 
* also moved handle and rank on same line
screen shot 2017-10-16 at 4 16 30 pm
screen shot 2017-10-16 at 4 16 03 pm

screen shot 2017-10-16 at 4 17 28 pm

**Comment by bonustrack** [comment]
Almost good. Can you change the breakpoint to `768px` `small`? 
And also can you update the style of the loader too? Now when the account is loading the frame is smaller than what expected in result:
loading

**Comment by jm90m** [comment]
@bonustrack alright changed break point and updated loader

screen shot 2017-10-16 at 4 52 58 pm
screen shot 2017-10-16 at 4 53 04 pm

**Comment by bonustrack** [comment]
Idealy it the loader look like this:

image

**Comment by jm90m** [comment]
@bonustrack ahh i see, should be centered now

screen shot 2017-10-16 at 5 25 45 pm

**Comment by bonustrack** [comment]
Can you make the space between element (in red) little bit bigger?
image
Also make the height of the frame bigger so load or not loaded it has the same size, now there is a difference see:
image
For width `768px` in CSS it would be better if we dont hardcode it but use this one https://github.com/busyorg/busy/blob/702bc7cb698fb8f200a768024cfbc0285258ad13/src/styles/modules/variables.less#L8

**Comment by jm90m** [comment]
@bonustrack alright made updates based on your suggestions, but we can't use the current variable `@small` on there, since it is `min-width 768px`; for this case we need it to be `max-width: 768px;`

**Comment by bonustrack** [comment]
The layout is broken when unlogged, everything appear in same line see:
image

Also it would be nice to keep the button "Follow" even unlogged. Later we would add a popup to login when click on "Follow" while unlogged

**Comment by jm90m** [comment]
@bonustrack I updated it to show follow button for unlogged user, that seems to fix layout issue, but also follow button currently does nothing.
* I will create ticket on trello & github to address unlogged follow button issue

**Comment by bonustrack** [comment]
@jm90m dont create ticket in Trello anymore, we will use only GitHub. But there is already issues for unlogged actions here:

https://github.com/busyorg/busy/issues/803
https://github.com/busyorg/busy/issues/469
https://github.com/busyorg/busy/issues/315

**Comment by Sekhmet** [comment]
I think that we should show Follow button for unlogged user as well as we will ideally show login modal after clicking on it.

**Comment by Sekhmet** [comment]
image
This is how it looks like at 900px width.

I think that we should add margin-left to avatar here.

**Comment by jm90m** [comment]
Added padding for medium / large screen widths, also updated User Header Loader for those screen sizes.

screen shot 2017-10-17 at 1 05 08 pm

**Comment by bonustrack** [comment]
@jm90m avatar should be aligned with the logo when window size is bigger than `998px`
image

**Comment by jm90m** [comment]
@bonustrack removed padding for window bigger than 998px

screen shot 2017-10-17 at 2 41 08 pm

**Comment by bonustrack** [comment]
Can we have same padding for window bigger than 998px for loading placeholder too? Then we will be perfect!

**Comment by jm90m** [comment]
@bonustrack correct me if I'm wrong but with my latest changes window does have same padding for loading placeholders. My latest changes affected `UserHeader__container` which the loading container is a part of:

screen shot 2017-10-17 at 3 03 26 pm

**Comment by bonustrack** [comment]
I mean in left here:
image

Also when loaded it's not aligned to logo
image

**Comment by jm90m** [comment]
@bonustrack ahh I see what you mean, looks like the width previously set was not the right length of the alignment, should be updated now!

screen shot 2017-10-17 at 3 45 45 pm
screen shot 2017-10-17 at 3 52 18 pm

**Comment by jm90m** [comment]
@bonustrack awesome thanks 🙏

**Pull request merged**